### DISCUSSION TITLE
Font is broken on my instance

### DISCUSSION THREAD

**Discussion by BradKML** [YU000jp/logseq-theme-classic-dark-theme-pack#10] [Q&A]
Most of the themes are stuck on Times New Roman (or some other Serif font) instead of using default fonts for each theme pack? What can be going wrong?
![image](https://github.com/YU000jp/logseq-theme-classic-dark-theme-pack/assets/58927531/4b2b9229-3605-450c-a11c-1fa09b413cb1)

**Comment by YU000jp** [comment]
@BrandonKMLee 
Please turn off the item "Unset 'font-family' in 'html' For fast font loading" in the plugin settings. It might be a bad thing for that item to be true by default. Sorry.

**Reply by BradKML** [reply]
Please address this well since I am somewhat new to themes, and that I am sure not every computer has sufficient fonts installed (Chocolatey installer)
